### PR TITLE
Travis reports error when it cannot find a wheel

### LINKED ISSUE #244: question about install
Hello!
I've been trying to install the InterOp Library and I seem to have run into some issues. 
I have independently tried both the PyPI package as well downloading and specifying the wheel file:

pip3 install interop
pip3 install interop-1.1.16-cp37-cp37m-macosx_10_15_x86_64.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl 

The install itself seems to work fine but I get the following error on testing:

python3 -m interop --test

Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/interop/py_interop_run.py", line 14, in swig_import_helper
    return importlib.import_module(mname)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 670, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 583, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 1043, in create_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
ImportError: dlopen(/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/interop/_py_interop_run.so, 2): Symbol not found: ____chkstk_darwin
  Referenced from: /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/interop/_py_interop_run.so
  Expected in: /usr/lib/libSystem.B.dylib
 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/interop/_py_interop_run.so

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/interop/__main__.py", line 4, in <module>
    from interop import py_interop_run,py_interop_metrics,py_interop_run_metrics,py_interop_comm,py_interop_table,py_interop_plot,py_interop_summary
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/interop/py_interop_run.py", line 17, in <module>
    _py_interop_run = swig_import_helper()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/interop/py_interop_run.py", line 16, in swig_import_helper
    return importlib.import_module('_py_interop_run')
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ModuleNotFoundError: No module named '_py_interop_run'

Before I go the more involved way of building from source; I wanted to check if you may have some obvious tips/suggestions for me that I may be missing.
Thanks in advance!
Vishal

### PR DESCRIPTION
**Pull request by ezralanglois**
Downgrading the OSX image to support older Macs
This caused brew to break. I tried several different things to fix this, in the end I found an environment variable that prevents brew from auto updating.

Resolves #244

### CODE REVIEW

**Comment by ezralanglois** [comment]
@nudpa This is ready

File: .travis.yml [hunk 1]
```diff
@@ -22,36 +22,68 @@ language: cpp
...
+        # `cache: ccache: true` has no effect if `language:` is not `c` or `cpp`
+        - $HOME/.ccache
+        # https://stackoverflow.com/questions/39930171/cache-brew-builds-with-travis-ci
+        - $HOME/Library/Caches/Homebrew
+        - /usr/local/Homebrew/
+        # used in OSX custom build script dealing with local bottle caching
+        - $HOME/local_bottle_metadata
     env: DEPLOY_BUILD=true SCRIPT="bash ./tools/package.sh $PWD $PWD/dist travis OFF Release Disable"
   - os: osx
-    osx_image: xcode12.2
```
Comment (ezralanglois) [inline .travis.yml:33]: Downgrade to support older images

File: tools/package.sh [hunk 2]
```diff
@@ -186,11 +186,12 @@ if [ "$PYTHON_VERSION" != "" ] && [ "$PYTHON_VERSION" != "Disable" ] && [ "$PYTH
       source $HOME/miniconda/etc/profile.d/conda.sh
       conda config --set channel_priority strict
       #conda update --all
+      echo "Found Miniconda"
     fi
     for py_ver in $python_versions; do
         echo "Building Python $py_ver - $CFLAGS"
-        if [ -e "/Users/bioinformatics/anaconda3" ]; then
```
Comment (ezralanglois) [inline tools/package.sh:192]: Fix switch to Anaconda. Not sure how this worked before

File: tools/prereqs/travis-osx-install.sh [hunk 3]
```diff
@@ -1,31 +1,44 @@
 #!/usr/bin/env bash
 
+echo "Install requirements using HomeBrew"
+export HOMEBREW_NO_INSTALL_CLEANUP=1
+export HOMEBREW_NO_AUTO_UPDATE=1
```
Comment (ezralanglois) [inline tools/prereqs/travis-osx-install.sh:5]: This is the magic change (one above may or may not help)

**Review by nudpa — APPROVED** [review]

**Pull request merged**